merlin/models/tf/transformers/block.py

```diff
@@ -91,7 +91,7 @@ def __init__(
             self.transformer = get_tf_main_layer(transformer)
         else:
             self.transformer = transformer
-
+        self.transformer.supports_masking = True
```

Comment:
set `supports_masking=True` so that the transformer layer can forward the `mask` tensor created by ` SequenceMaskLastInference`